src/scribe_data/utils.py

```diff
@@ -598,14 +596,15 @@ def list_languages_with_metadata_for_data_type(language_metadata=_languages):
         # Check if there are sub-languages.
         if "sub_languages" in lang_data:
             # Add the sub-languages to current_languages with metadata.
-            for sub_key, sub_data in lang_data["sub_languages"].items():
-                current_languages.append(
-                    {
-                        "name": f"{lang_data.get('name', lang_key)}/{sub_data.get('name', sub_key)}",
-                        "iso": sub_data.get("iso", ""),
-                        "qid": sub_data.get("qid", ""),
-                    }
-                )
+            current_languages.extend(
```

Comment:
We can use `extend` here to avoid the `for`/`append` loop and clean up the code a bit :)